app/services/proofing/lexis_nexis/ddp/proofer.rb

```diff
@@ -37,6 +37,7 @@ def proof_applicant(applicant, result)
 
         def process_response(response, result)
           body = response.response_body
+          result.response_body = body
           result.transaction_id = body['request_id']
```

Comment:
ok so here... `body` is a parsed JSON structure (hence why we can do `body['request_id']` ... I think this means the redactor takes in a struct and not a plain string